protobuf/state/state.proto

```diff
@@ -22,8 +63,23 @@ message ConnectionStatus {
     bool is_mesh_peer = 7;
 }
 
+enum LoginEventType {
+    LOGIN = 0;
+    LOGOUT = 1;
+}
+
+message LoginEvent {
+    LoginEventType type = 1;
+}
+
 message AppState {
     oneof state {
         ConnectionStatus connection_status = 1;
+        SettingChange setting_change = 2;
```

Comment:
Meshnet state to be added later?

Reply:
For this task there's a notification for meshnet setting change(on/off). I don't see any task for more complicated meshnet data, such as new peers appearing or changing permissions, maybe it should be added. :thinking: